roseau/load_flow/models/buses.py

```diff
@@ -253,6 +253,29 @@ def propagate_limits(self, force: bool = False) -> None:
             bus._min_voltage = self._min_voltage
             bus._max_voltage = self._max_voltage
 
+    def find_neighbors(self) -> Iterator[Id]:
+        """Find the buses connected to this bus via a line or switch recursively."""
+        from roseau.load_flow.models.lines import Line, Switch
+
```

Comment:
I'm wondering if "neighbors" is not a bit confusing. When I first saw this method, I thought it would only give the adjacent buses connected to that bus.

Reply:
Right. I was going with `get_connnected_buses` at first but it suffers from the same ambiguity as `find_neighbors` in that it could mean buses connected with one line or switch. Do you have a better name? I can also clarify the docstring to say something like `Find all buses connected to this bus via one or more lines or switches, stop at transformers.`

Reply:
I think we've used the term `galvanic component` for this kind of stuff, but I'm not convinced that it would be understood easily. Between `get_connected_buses` and `find_neighbors`, I think I prefer the former as I find it less confusing, but I agree that there is still an ambiguity that has to be clarified in the docstring.

Reply:
OK I renamed it to `get_connected_buses` and clarified that we are talking about "galvanically connected" buses.